CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ## Unreleased
 
 * Added the flag '-d', '--dependency' to **find-dependencies** command to get the content items that cause the dependencies between two packs.
-
+* Fixed a bug for list uploading.
```

Comment:
Please elaborate, something like 
`* Fixed an issue in **Upload** where List items were not uploaded when < reason >`